docs/book/requirements.rst

```diff
@@ -0,0 +1,88 @@
+.. index::
+    single: System Requirements
+
+System Requirements
+===================
+
+Here you will find the list of system requirements that have to be adhered to be able to use **Sylius**.
+First of all have a look at the `requirements for running Symfony <http://symfony.com/doc/current/reference/requirements.html>`_.
+
+Read about the `LAMP stack <https://en.wikipedia.org/wiki/LAMP_(software_bundle)>`_.
+
+Operating Systems
+~~~~~~~~~~~~~~~~~
+
+The recommended operating systems for running Sylius are the Unix systems - **Linux, MacOS**.
+
+Web server and configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We do recommend using Apache web server ≥ 2.2.
+Remember to load the ``php`` module in the ``/etc/apache2/httpd.conf``.
+
+PHP required modules and configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Required**:
+
++---------------+-----------------------+
+| PHP           | ^5.6                  |
++---------------+-----------------------+
+| ext-gd        |                       |
++---------------+-----------------------+
+| ext-exif      |                       |
++---------------+-----------------------+
+| ext-fileinfo  |                       |
++---------------+-----------------------+
+| php5-intl     |                       |
++---------------+-----------------------+
+| memory-limit  | ≥512M                 |
++---------------+-----------------------+
+| date.timezone | Europe/Warsaw         |
++---------------+-----------------------+
+
+.. note::
+
+    Use your local timezone, for example America/Los_Angeles or Europe/Berlin. See http://www.php.net/timezones for the list of all available timezones.
+
+**Recommended**:
+
++---------------+-----------------------+
+| PHP           | ^7.0                  |
++---------------+-----------------------+
+| ext-gd        |                       |
++---------------+-----------------------+
+| ext-exif      |                       |
++---------------+-----------------------+
+| ext-fileinfo  |                       |
++---------------+-----------------------+
+| php5-intl     |                       |
++---------------+-----------------------+
+| php70-xdebug  |                       |
```

Comment:
Anyway, it shouldn't be here in the requirements section, it's only suitable for development.